api_fetch.js

```diff
@@ -328,4 +369,4 @@ function searchTags(tag) {
       noMatchError.innerHTML = "No matching challenges";
     }
   }
-}
+}
```

Comment:
Here, the only change that has happend is related to whitespace. In the pre-existing version, there was a final newline character (i.e. an "empty line") after the last curly brace. In your updated version, there is none.

Different editors (and differently configured instances of the same editor) will handle things like this differently. In a team, it's always good to agree on a specific practice and configure editors (or other tools) to stick to that practice.